src/components/RoomSelector.vue

```diff
@@ -142,20 +151,34 @@ export default {
 				return roomsTemp.filter(room => room.displayName.toLowerCase().includes(this.searchText.toLowerCase()))
 			}
 		},
+
+		noMatchFoundTitle() {
+			return this.listOpenConversations
+				? t('spreed', 'No open conversations found')
+				: t('spreed', 'No conversations found')
+		},
+
+		noMatchFoundSubtitle() {
+			return this.listOpenConversations
+				? t('spreed', 'Either there are no open conversations or you joined all of them.')
+				: t('spreed', 'Check spelling or use complete words.')
+		},
 	},
-	beforeMount() {
-		this.fetchRooms()
+	async beforeMount() {
```

Comment:
Not sure that we should change this hook to async;
it's not blocking modal from rendering, but makes the rest code about `$store` async.
I'm afraid that it could break usage of component in other places, where store should be available beforehand (Or maybe it'll just force computed property `availableRooms` to recalculate, but still - not what we want)
And `fetchRooms()` is asyncronous by itself, so other code should be fine
```suggestion
	beforeMount() {
```